### PR TITLE
allow loading external registry deps

### PR DESCRIPTION
**Pull request by colinxs**
This allows the UUIDs tested to be loaded from external registries instead of just the registry being tested.

### CODE REVIEW

**Comment by DilumAluthge** [comment]
@fredrikekre I think that you wrote this code originally. Could you review?

**Comment by DilumAluthge** [comment]
Bump @fredrikekre

File: src/RegistryCI.jl [hunk 1]
```diff
@@ -84,4 +86,23 @@ function test(path=pwd())
...
+    for reg in registry_deps
+        extrauuids = extrauuids ∪ load_registry_uuids(reg)
+    end
+    extrauuids
+end
+
+function load_registry_uuids(registry::AbstractString)
+    mktempdir() do tmp
+        cd(tmp) do
+            run(`git clone $registry $(pwd())`)
```
Comment (fredrikekre) [inline src/RegistryCI.jl]: How about Pkg.Registry.add instead?

**Comment by DilumAluthge** [comment]
Bump @colinxs 

Can you make the changes that @fredrikekre requested?

**Comment by DilumAluthge** [comment]
Bump @colinxs

Can you please make the changes that @fredrikekre requested?

**Comment by DilumAluthge** [comment]
bors ping

**Comment by BenJWard** [comment]
I think this solves the issue I'm having using RegistryCI over at the BioJuliaRegistry?

https://travis-ci.com/BioJulia/BioJuliaRegistry/jobs/272516827

**Comment by BenJWard** [comment]
Would it be acceptable for me to take the current progress and make the change requested by @fredrikekre?

**Comment by DilumAluthge** [comment]
> Would it be acceptable for me to take the current progress and make the change requested by @fredrikekre?

Absolutely! You’ll probably need to rebase against master.

**Comment by DilumAluthge** [comment]
It’ll probably be easiest for you to make a brand new PR.

**Comment by BenJWard** [comment]
Ok I've submitted at #194

**Pull request closed without merging**